apps/web/src/app/(private)/traces/_components/TracesTable/SearchBox/constants.ts

```diff
@@ -0,0 +1,37 @@
+import { SearchColumn, Operator } from '../types'
+
+export const TIME_OPERATORS: Operator[] = [
+  { label: 'is', value: 'eq' },
+  { label: 'before', value: 'lt' },
+  { label: 'after', value: 'gt' },
+]
```

Comment:
Maybe you could use the new fancy date picker instead. For date picking, the date picker is nicer

Reply:
yeah i will implement this pretty soon